packages/block-editor/src/components/block-list/block-list-item.native.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { ReadableContentView } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import BlockListBlock from './block';
+import BlockInsertionPoint from './insertion-point';
+
+const stretchStyle = {
+	flex: 1,
+};
+
+export class BlockListItem extends Component {
+	render() {
+		const {
+			clientId,
+			isReadOnly,
+			shouldShowInsertionPointBefore,
+			shouldShowInsertionPointAfter,
+			contentResizeMode,
+			shouldShowInnerBlockAppender,
+			...restProps
+		} = this.props;
+		const readableContentViewStyle =
+			contentResizeMode === 'stretch' && stretchStyle;
+		return (
+			<ReadableContentView style={ readableContentViewStyle }>
+				<View
+					style={ readableContentViewStyle }
+					pointerEvents={ isReadOnly ? 'box-only' : 'auto' }
+				>
+					{ shouldShowInsertionPointBefore && (
+						<BlockInsertionPoint />
+					) }
+					<BlockListBlock
+						key={ clientId }
+						showTitle={ false }
+						clientId={ clientId }
+						{ ...restProps }
+					/>
+					{ ! shouldShowInnerBlockAppender() &&
+						shouldShowInsertionPointAfter && (
```

Comment:
It's not clear to me why we need the `! shouldShowInnerBlockAppender()` check here. I assume we need this because there is a scenario where `shouldShowInnerBlockAppender()` and `shouldShowInsertionPointAfter` are both `true`, but I had some trouble finding that scenario. Could you share what that scenario is (or if there's a different reason we need to check `! shouldShowInnerBlockAppender()` here)?

Reply:
`shouldShowInnerBlockAppender()` and `shouldShowInsertionPointAfter` are both `true` for the last block in inner blocks when we press on the list appender
![appenderr](https://user-images.githubusercontent.com/16336501/82260064-e9656a00-995c-11ea-9aaa-7437599596ef.gif)

Reply:
I was trying to recreate it with the column block and couldn't seem to do it. Thanks for showing me!